xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/NfTokenCancelOffer.java

```diff
@@ -42,7 +42,7 @@
    *
    * @return Array of TokenIDs, each identifying a unique NfToken object, to cancel the offers for.
    */
-  @JsonProperty("TokenIDs")
+  @JsonProperty("TokenOffers")
```

Comment:
@mukulljangid Can we add unit tests for this case? E.g., we should have some unit tests that validate JSON.